samples/benchmarks/coremark/boards/Kconfig.sysbuild_boards

```
@@ -0,0 +1,16 @@
+#
+# Copyright (c) 2024 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+if $(BOARD) = "nrf5340dk_nrf5340_cpuapp"
+
+config APP_NRF53_CPUNET_RUN
```

To make it conssitent with nRF53 Kconfig options naming conventions like `CONFIG_BOARD_ENABLE_CPUNET`

```suggestion
config APP_CPUNET_RUN
```